packages/firebase_core/pubspec.yaml

```diff
@@ -0,0 +1,23 @@
+name: firebase_core
+description: Firebase Core plugin for Flutter.
+author: Flutter Team <flutter-dev@googlegroups.com>
+homepage: https://github.com/flutter/plugins/tree/master/packages/firebase_core
+version: 0.0.1
+
+flutter:
+  plugin:
+    androidPackage: io.flutter.plugins.firebase.core
+    iosPrefix: FLT
+    pluginClass: FirebaseCorePlugin
+
+dependencies:
+  flutter:
+    sdk: flutter
+  meta: "^1.0.5"
```

Comment:
random/feel-free-to-ignore: what do the quotes do?

Reply:
I don't think there's any particular reason why this needs to be here, it's copy-pasted from elsewhere. Quotes are more important with version constraints (`>`). https://www.dartlang.org/tools/pub/dependencies#version-constraints